genetic_algorithms/algorithm_wrappers/visualizations/__init__.py

```diff
@@ -1,2 +1,3 @@
 from genetic_algorithms.algorithm_wrappers.visualizations.visualization_wrapper import VisualizationSubscriber
 from genetic_algorithms.algorithm_wrappers.visualizations.traveling_salesman_visualization import TravelingSalesmanVisualization
+from genetic_algorithms.algorithm_wrappers.visualizations.graph_coloring_visualization import GraphColoringVisualization
```

Comment:
@hankaj 
This line is important when adding new visualizations, because thanks to it they could be injected in as subsribers in CLI